### PR TITLE
feat(Linkify): Add markdown support

### PR DESCRIPTION
**Pull request by jlemm45**
**Summary:**
Adds markdown support. I've experimented with several solutions so far and `react-markdown` got me closest to what I was trying to accomplish. With `react-markdown` the links are still parsed as before as well as the additional markdown. 

Still a WIP.

### CODE REVIEW

File: src/Linkify/Linkify.js [hunk 1]
```diff
@@ -1,49 +1,32 @@
...
-const URL_REGEX = /(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#/%?=~_|!:,.;]*[-A-Z0-9+&@#/%=~_|])/gi;
-
-const Link = ({ href, linkStyle }) => (
-  <a
-    target="_blank"
-    rel="noopener noreferrer"
-    style={{
-      color: 'inherit',
-      textDecoration: 'underline',
-      ...linkStyle,
```
Comment (choochootrain) [inline src/Linkify/Linkify.js]: do we need to preserve the `linkStyle` prop?

**Comment by jlemm45** [comment]
@choochootrain Added

**Comment by kylealwyn** [comment]
screen shot 2019-01-15 at 11 10 18 am

Looks like the example is broken. We may need to recursively flatten down Linkify's children.

**Comment by jlemm45** [comment]
@kylealwyn what is the use case for nesting the `Linkify` component?

**Pull request merged**